snap/snapcraft.yaml

```diff
@@ -128,19 +139,30 @@ parts:
     - on arm64: http://ports.ubuntu.com/ubuntu-ports/pool/main/i/icu/libicu60_60.2-3ubuntu3_arm64.deb
     source-type: deb
 
+  libssl:
+    plugin: dump
+    source:
+    - on amd64: http://archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.0g-2ubuntu4.1_amd64.deb
+    - on i386: http://archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.0g-2ubuntu4.1_i386.deb
+    - on armhf: http://ports.ubuntu.com/ubuntu-ports/pool/main/o/openssl/libssl1.1_1.1.0g-2ubuntu4.1_arm64.deb
+    - on arm64: http://ports.ubuntu.com/ubuntu-ports/pool/main/o/openssl/libssl1.1_1.1.0g-2ubuntu4.1_arm64.deb
+    source-type: deb
```

Comment:
Hmm why this? Doesn't core provide `libssl` for security reasons?

Reply:
I was focused on core16, which provides libssl1.0 only, so I added this. Core18 does provide libssl1.1 though. We could move to that..